docs/nlu/choosing-a-pipeline.rst

```diff
@@ -154,12 +158,12 @@ final context dictionary is used to persist the model's metadata.
 
 
 
-Returned Entities Object
-------------------------
-In the object returned after parsing there are two fields that show information
-about how the pipeline impacted the entities returned. The ``extractor`` field
-of an entity tells you which entity extractor found this particular entity.
-The ``processors`` field contains the name of components that altered this
+Reading an "entity" object
```

Comment:
I agree that the quotes are needed here 👍!

Regarding the rewording, when I first read it, I thought it might be about how to read the entities object from a program (rather than "How to interpret the returned entities object"). How about `The "entity" object explained"` or `Interpreting the "entity" object`?